helm/regclient/values.yaml

```diff
@@ -293,8 +303,8 @@ volumePermissions:
   enabled: false
   image:
     registry: docker.io
-    repository: bitnami/bitnami-shell
-    tag: "10"
+    repository: mosipid/os-shell
+    tag: "12-debian-12-r46"
     pullPolicy: Always
     ## Optionally specify an array of imagePullSecrets.
     ## Secrets must be manually created in the namespace.
```

Comment:
repository: mosipid/registration-client
to repository: mosipqa/registration-client
t